include/nccl_ofi_rdma.h

```diff
@@ -88,8 +88,10 @@ typedef struct nccl_net_ofi_rdma_ctrl_fl_item {
 } nccl_net_ofi_rdma_ctrl_fl_item_t;
 
 /* Structure used to store bounce buffers in a free list */
+#define BOUNCE_BUFFER_ALIGNMENT 128
 typedef struct nccl_net_ofi_rdma_bounce_fl_item {
 	nccl_ofi_freelist_reginfo_t fl_reginfo;
+	char padding[BOUNCE_BUFFER_ALIGNMENT - (sizeof(nccl_ofi_freelist_reginfo_t) % BOUNCE_BUFFER_ALIGNMENT)];
```

Comment:
If `sizeof(nccl_ofi_freelist_reginfo_t) % BOUNCE_BUFFER_ALIGNMENT == 0`, then there ought to be 0 padding, not `BOUNCE_BUFFER_ALIGNMENT` padding.